src/Message/DefaultApiPlatformMessage.php

```diff
@@ -138,6 +138,11 @@ public static function __apiPlatformController(): string
         return PlaceholderAction::class;
     }
 
+    public static function __stateless(): bool
+    {
+        return true;
```

Comment:
This can also be null, but since we are using this as an api, true is also a sane default